### PR TITLE
feat: Add Past filter to Explore page

### LINKED ISSUE #5802: Search: Allow search of "Past" events and add the option "Past" into the filters
The search only seems to allow searching future events. Please add the option "Past" as a filter.

Screenshot from 2020-11-28 01-18-13

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5802 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
For past events, please sort them by date in descending order. So, latest event should be on top

**Comment by mariobehling** [comment]
Past filter should not be an exclusive filter. It should be possible to combine it with any other date filter, e.g. Today and Past.

**Comment by iamareebjamal** [comment]
@mariobehling The current implementation for all date filters is exclusive already. Making it compound is significantly more work and should be handled in another PR or else even this issue will be blocked.

**Comment by iamareebjamal** [comment]
Opened #5977 for the above feature

**Review by mariobehling — APPROVED** [review]

**Comment by iamareebjamal** [comment]
@sachinchauhan2889 Update?

**Comment by sachinchauhan2889** [comment]
> @sachinchauhan2889 Update?

Sir,  work in progress. Commit in next hour

File: app/routes/explore.js [hunk 1]
```diff
@@ -180,26 +181,43 @@ export default class ExploreRoute extends Route {
...
-            val  : new Date()
-          },
-          {
-            name : 'ends-at',
-            op   : 'ge',
-            val  : new Date()
-          }
-        ]
-      });
+      if (params.is_past) {
```
Comment (iamareebjamal) [inline app/routes/explore.js:184]: Put it in else if

**Pull request merged**